addons/a11y/src/components/Tabs.tsx

```diff
@@ -35,7 +34,7 @@ const Item = styled.button(
       borderBottom: `3px solid ${theme.color.secondary}`,
     },
   }),
-  ({ active, theme }) =>
+  ({ active, theme }: any) =>
```

Comment:
we can at least say active is boolean right?

Reply:
Unfortunately same as https://github.com/storybooks/storybook/pull/5738#discussion_r260462864 :/ 

So I will take a look to `@storybook/theming` really fast.

Reply:
Even just `{active:boolean, theme:any}` would be stricter, does that also error?

Reply:
Yes as we are destructuring an object with other properties, I also tried `{active:boolean; theme:any; [key: string]: any}` but still get TS errors... I can not do better than `any` for now